src/Global.js

```diff
@@ -29,7 +29,7 @@
  */
 
 // runtime check for already included Konva
-(function(){
+(function(global){
```

Comment:
Global detection already included in https://github.com/hugomrdias/konva/blob/node/src/Global.js#L203